src/lib/spinel/spinel.h

```diff
@@ -4946,6 +4946,21 @@ enum
 
     SPINEL_PROP_MULTIPAN__END = 0x910,
 
+    SPINEL_PROP_INFRA_IF__BEGIN = 0x910,
+
+    /// Infrastructure interface setup.
+    /** Format: `LbA(6)`
+     * Type: Write
+     *
+     * `L`: The infrastructure interface index.
+     * `b`: If the infrastrue interface is running.
+     * `A(6)`: The IPv6 addresses of the infrastructure interface.
```

Comment:
The code parsing this property seems to decode it as array of structs where each struct contains an IPv6 address. Should we change this to `A(t(6))` to be match the decoding logic

Or we can consider changing the decoding logic so it treats it as `A(6)`?

Reply:
Updated the decoding logic and treat it as `A(6)` since that's a little bit more efficient.